frontend/src/organizations/OrganizationEditContainer.js

```diff
@@ -3,229 +3,272 @@
 
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import PropTypes from 'prop-types'
-import { bindActionCreators } from 'redux'
-
-import { addOrganization, getOrganization, updateOrganization } from '../actions/organizationActions'
-import { getUpdatedLoggedInUser } from '../actions/userActions'
-import Loading from '../app/components/Loading'
-import OrganizationEditForm from './components/OrganizationEditForm'
-import toastr from '../utils/toastr'
-import ORGANIZATION from '../constants/routes/Organizations'
-import Modal from '../app/components/Modal'
-import PERMISSIONS_ORGANIZATIONS from '../constants/permissions/Organizations'
-import { withRouter } from '../utils/withRouter'
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+
+import {
+  addOrganization,
+  getOrganization,
+  updateOrganization,
+} from "../actions/organizationActions";
+import { getUpdatedLoggedInUser } from "../actions/userActions";
+import Loading from "../app/components/Loading";
+import OrganizationEditForm from "./components/OrganizationEditForm";
+import toastr from "../utils/toastr";
+import ORGANIZATION from "../constants/routes/Organizations";
+import Modal from "../app/components/Modal";
+import PERMISSIONS_ORGANIZATIONS from "../constants/permissions/Organizations";
+import { withRouter } from "../utils/withRouter";
 
 class OrganizationEditContainer extends Component {
-  constructor (props) {
-    super(props)
+  constructor(props) {
+    super(props);
 
     this.state = {
       fields: {
-        name: '',
-        addressLine_1: '',
-        addressLine_2: '',
-        addressLine_3: '',
-        city: '',
-        postalCode: '',
-        state: '',
-        county: '',
-        country: '',
-        type: 2,
-        actionsType: 1,
-        status: 1
-      }
-    }
-
-    this.submitted = false
-
-    this._handleInputChange = this._handleInputChange.bind(this)
-    this._handleCreate = this._handleCreate.bind(this)
-    this._handleUpdate = this._handleUpdate.bind(this)
+        org_name: "",
+        org_addressLine1: "",
+        org_addressLine2: "",
+        org_city: "",
+        org_postalCode: "",
+        org_state: "",
+        org_country: "",
+        org_type: 2,
+        org_actionsType: 1,
+        org_status: 1,
+        att_representativeName: "",
+        att_streetAddress: "",
+        att_otherAddress: "",
+        att_city: "",
+        att_province: "",
+        att_country: "",
+        att_postalCode: "",
+      },
+    };
+
+    this.submitted = false;
+
+    this._handleInputChange = this._handleInputChange.bind(this);
+    this._handleCreate = this._handleCreate.bind(this);
+    this._handleUpdate = this._handleUpdate.bind(this);
   }
 
-  componentDidMount () {
-    if (this.props.mode === 'add') {
-      return
+  componentDidMount() {
+    if (this.props.mode === "add") {
```

Comment:
We are using JS Standard linting which doesn't use semi-colons. Try running npm run lint to see any required changes.

Reply:
i did ran "npm run lint" and it didn't complain about semi-colon.

Reply:
i will take that previous statement back. npm run lint fixed semicolon and i have pushed that changes.